### ISSUE TITLE
Why not RenderOptions.BitmapScalingMode="NearestNeighbor" like wpf?

### ISSUE THREAD

**Issue by suweitao8** [dotnet/maui#21752]
### Description

Why not RenderOptions.BitmapScalingMode="NearestNeighbor" like wpf?

### Public API Changes

add RenderOptions.BitmapScalingMode to Image 

### Intended Use-Case

pixel image display

**Comment by Redth** [comment]
Are you able to elaborate on this?

I think you are asking if we would add this API to the MAUI Image control? Is that an accurate understanding?  What problem is this trying to solve specifically?  Do you have examples? samples?

I do not believe this would be feasible to implement across platforms in a way that adds value to the product.

**Comment by suweitao8** [comment]
> Are you able to elaborate on this?
> 
> I think you are asking if we would add this API to the MAUI Image control? Is that an accurate understanding? What problem is this trying to solve specifically? Do you have examples? samples?
> 
> I do not believe this would be feasible to implement across platforms in a way that adds value to the product.

Yes, I require this parameter to achieve a pixel art effect. I need to upscale a large pixel art image. While SKCanvasView does work for this purpose, it performs extremely poorly when scaling up large images. Similar to creating a pixel art drawing software for Windows.
